CWE-94-Python_darwin/mit-han-lab_neurips-micronet/u.py

```
@@ -0,0 +1,944 @@
+                    pass
+            kwargs[splits[0]] = v
+
+        return cls(args.res).load().var(**kwargs).save()
+
+    @classmethod
+    def clean(cls, *directories):
+        configs = cls.load_all(*directories)
+        for config in configs:
+            if not (config.train_results.exists() or len(config.models.ls()[1]) > 0):
+                config.res.rm()
+                self.log('Removed %s' % config.res)
```

Ruff (0.13.2)

Undefined name `self`

(F821)